Src/Newtonsoft.Json.UnityConverters.Tests/ConvertingUnityTypes/QuaternionTests.cs

```diff
@@ -0,0 +1,15 @@
+﻿using UnityEngine;
+
+namespace Newtonsoft.Json.UnityConverters.Tests.ConvertingUnityTypes
+{
+    public class QuaternionTests : TypeTester<Quaternion>
+    {
+        public static (Quaternion deserialized, string serialized)[] representations = new[] {
+            (new Quaternion(1, 2, 3, 4), @"{""x"":1.0,""y"":2.0,""z"":3.0,""w"":4.0}")
+        };
+
+        protected override void ConfigureSettings(JsonSerializerSettings settings)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add a nested comment explaining why this method is empty, throw a 'NotSupportedException' or complete the implementation.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=4930242)